File: product-web-app/service/bandValidator.ts
import Band from '../model/band.js';

export default class BandValidator {
  validateBand(band: Band) {
    if (band.level < 0) {
      return 'Level of band should be greater than 0!';
    }

    if (band.level > 10) {
      return 'Level of band should be smaller than 10!';
    }

    return null;
  }
}

File: product-backend/src/main/java/org/kainos/ea/service/BandValidator.java
package org.kainos.ea.service;

import org.kainos.ea.model.BandRequest;

public class BandValidator {

    public String isValidBand(BandRequest band) {

        if (band.getLevel() < 0 || band.getLevel() > 9){
            return "Band level out of boundaries 0-9";
        }

        return null;
    }
}
